### PR TITLE
Custom Job Titles

### PR DESCRIPTION
**Pull request by sheepishgoat**
## About The Pull Request
Adds in custom job titles. This is a upstream version of this PR on our downstream repo: https://github.com/sheepishgoat/GS13-Citadel/pull/16

## Why It's Good For The Game
This helps encourage roleplay by letting people choose job titles closer to what their character actually does.

## Changelog
:cl:
add: Adds in custom job titles.
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_titles.dm [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	alt_titles = list("Gardener", "Herbalist", "Botanical Researcher", "Hydroponicist", "Farmer", "Beekeeper", "Vintner")
+
+/datum/job/curator
+	alt_titles = list("Journalist", "Librarian", "Keeper")
+
+/datum/job/chaplain
+	alt_titles = list("Priest", "Priestess", "Prior", "Monk", "Nun", "Counselor")
+
+/datum/job/janitor
+	alt_titles = list("Custodian", "Sanitation Technician", "Maid", "Trash Can", "Disposal Unit")
```
Comment (deathride58) [inline code/modules/jobs/job_titles.dm:30]: >"Trash Can"

How do I approve a PR multiple times

File: code/modules/jobs/job_titles.dm [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+/datum/job/chemist
+	alt_titles = list("Pharmacist", "Pharmacologist")
+
+/datum/job/virologist
+	alt_titles = list("Microbiologist", "Biochemist", "Pathologist")
+
+/datum/job/geneticist
+	alt_titles = list("Gene Therapist", "Genetics Researcher")
+
+/datum/job/psychologist
```
Comment (deathride58) [inline code/modules/jobs/job_titles.dm]: Psychologists do not exist (could probably roll into doctor itself)

File: code/modules/jobs/job_titles.dm [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+	alt_titles = list("Therapist", "Psychiatrist")
+
+/datum/job/junior_doctor
```
Comment (deathride58) [inline code/modules/jobs/job_titles.dm]: Junior doctors do not exist

File: code/modules/jobs/job_titles.dm [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+/datum/job/rd
+	alt_titles = list("Research Manager", "Science Administrator")
+
+/datum/job/scientist
+	alt_titles = list("Circuitry Designer", "Xenobiologist", "Xenobotanist", "Xenoarcheologist", "Chemical Researcher", "Researcher", "Pyrotechnician")
+
+/datum/job/roboticist
+	alt_titles = list("Biomechanical Engineer", "Mechatronic Engineer", "Mechanic")
+
+/datum/job/junior_scientist
```
Comment (deathride58) [inline code/modules/jobs/job_titles.dm]: Junior Scientists do not exist

File: code/modules/jobs/job_titles.dm [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+/datum/job/chief_engineer
+	alt_titles = list("Head Engineer", "Construction Coordinator", "Project Manager", "Power Plant Director")
+
+/datum/job/engineer
+	alt_titles = list("Maintenance Technician", "Engine Technician", "Electrician", "Structural Engineer", "Mechanic",  "Station Architect", "Nuclear Plant Operator")
+
+/datum/job/atmos
+	alt_titles = list("Firefighter", "Life Support Specialist", "Disposals Technician")
+
+/datum/job/junior_engineer
```
Comment (deathride58) [inline code/modules/jobs/job_titles.dm]: Junior engineers do not exist

File: code/modules/jobs/job_titles.dm [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+/datum/job/warden
+	alt_titles = list("Prison Chief", "Armory Manager", "Prison Administrator", "Brig Superintendent")
+
+/datum/job/officer
+	alt_titles = list("Security Agent", "Probation Officer", "Security Peacekeeper", "Security Guard", "Guardsman", "Security Cadet")
+
+/datum/job/detective
+	alt_titles = list("Forensics Technician", "Private Investigator", "Gumshoe")
+
+/datum/job/junior_officer
```
Comment (deathride58) [inline code/modules/jobs/job_titles.dm]: Junior officers do not exist

**Review by deathride58 — CHANGES_REQUESTED** [review]
Code's good! Just a few porting artifacts left over

**Review by deathride58 — APPROVED** [review]

**Comment by SandPoot** [comment]
I am not fond of how this PR goes stuff that could end up in issues
I'd rather have sandstorm's version be ported over as it can support special behaviors without being intrusive

**Pull request merged**